pkg/minikube/bootstrapper/bsutil/template/template.go

```diff
@@ -151,6 +151,56 @@ evictionHard:
   imagefs.available: "0%"
 `))
 
+// KubeAdmConfigTmplV1Beta2 is for Kubernetes v1.17+
+var KubeAdmConfigTmplV1Beta2 = template.Must(template.New("configTmpl-v1beta2").Funcs(template.FuncMap{
+	"printMapInOrder": printMapInOrder,
+}).Parse(`apiVersion: kubeadm.k8s.io/v1beta2
+bootstrapTokens:
+ - groups:
+    - system:bootstrappers:kubeadm:default-node-token
+   ttl: 24h0m0s
+   usages:
+    - signing
+    - authentication
+kind: InitConfiguration
+localAPIEndpoint:
+  advertiseAddress: {{.AdvertiseAddress}}
+  bindPort: {{.APIServerPort}}
+nodeRegistration:
+  criSocket: {{if .CRISocket}}{{.CRISocket}}{{else}}/var/run/dockershim.sock{{end}}
+  name: {{.NodeName}}
+  taints: []
+---
+{{ if .ImageRepository}}imageRepository: {{.ImageRepository}}
+{{end}}{{range .ExtraArgs}}{{.Component}}:
+  extraArgs:
+{{- range $i, $val := printMapInOrder .Options ": " }}
+    {{$val}}
+{{- end}}
+{{end -}}
+{{if .FeatureArgs}}featureGates:
+{{range $i, $val := .FeatureArgs}}{{$i}}: {{$val}}
+{{end -}}{{end -}}
+  timeoutForControlPlane: 4m0s
+apiVersion: kubeadm.k8s.io/v1beta2
+certificatesDir: {{.CertDir}}
+clusterName: kubernetes
+controlPlaneEndpoint: localhost:{{.APIServerPort}}
+controllerManager: {}
+dns:
+  type: CoreDNS
+etcd:
+  local:
+    dataDir: {{.EtcdDataDir}}
+imageRepository: k8s.gcr.io
+kind: ClusterConfiguration
+kubernetesVersion: {{.KubernetesVersion}}
+networking:
+  dnsDomain: {{if .DNSDomain}}{{.DNSDomain}}{{else}}cluster.local{{end}}
+  serviceSubnet: {{.ServiceCIDR}}
+scheduler: {}
+`))
+
```

Comment:
`kind: KubeletConfiguration` doesn't exist here.
Is it OK?
Or `kind: KubeletConfiguration`  isn't needed in Kubernetes v1.17+ ?

Reply:
@atoato88 the new config file `v1beta2.yaml` does not have `kind: KubeletConfiguration`, you can check the new file [here](https://gist.github.com/RubenBaez/9eb3d186d7abee4a98fa8c5e45352dae)

Reply:
@atoato88 I was reviewing a little more detail and found that `kind: KubeletConfiguration` is necessary, so I wrote it.

Reply:
@RubenBaez 
Thank you to response :+1: